.github/workflows/deployFreighterApiBeta.yml

```diff
@@ -19,6 +19,8 @@ jobs:
         uses: actions/checkout@v4
         with:
           ref: release/${{ github.event.inputs.release }}
+      - name: Enable Corepack
```

Comment:
The install process for latest versions of Yarn, so we have to run this cmd now rather than running `npm -g yarn`

https://yarnpkg.com/migration/guide#migration-steps